internal/boot/bootstrap_termus.go

```diff
@@ -0,0 +1,59 @@
+//go:build termus
+// +build termus
+
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package boot
+
+import (
+	"errors"
+	"os"
+)
+
+var Getpid = os.Getpid
```

Comment:
Nit: this shouldn't be capitalized/exported. Perhaps call it `getpid` or `osGetpid`. (it's an internal package, so it doesn't matter much, but we might as well be consistent).